.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ on:
       - master
 
 env:
-  NEXTJS_SOURCE_FILES: "src styles"
+  NEXTJS_SOURCE_FILES: "app styles"
```

Comment:
Isn't the support for /app still in beta?
https://nextjs.org/docs/basic-features/pages

Reply:
> Isn't the support for /app still in beta?
> https://nextjs.org/docs/basic-features/pages

Yes, we should decide whether to take this "risk" and start building our app with this experimental function instead of migrating it when they release it in stable.

Reply:
OK, zatím to můžeme nechat takto